frontend/awx/resources/inventories/InventoryPage/InventoryGroups.cy.tsx

```diff
@@ -0,0 +1,112 @@
+import { InventoryGroups } from './InventoryGroups';
+import { AwxGroup } from '../../../interfaces/InventoryGroup';
+
+describe('InventoryGroups', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: 'GET',
+        url: '/api/v2/inventories/**/groups?order_by=name&page=1&page_size=10',
+        hostname: 'localhost',
+      },
+      {
+        fixture: 'groups.json',
+      }
+    );
+    cy.intercept(
+      {
+        method: 'OPTIONS',
+        url: '/api/v2/inventories/**/ad_hoc_commands',
+        hostname: 'localhost',
+      },
+      {
+        fixture: 'ad_hoc_commands.json',
+      }
+    );
+  });
+
+  it('renders invetory groups list', () => {
+    cy.mount(<InventoryGroups />);
+    cy.fixture('groups.json')
+      .its('results')
+      .should('be.an', 'array')
+      .then((results: AwxGroup[]) => {
+        const group = results[0];
+        cy.contains(group.name);
+      });
+  });
+
+  it('deletes group from toolbar menu', () => {
```

Comment:
See comment above about deleting Inventory Groups.  You can keep this test if you want for now, but when that other work gets done this test should be rewritten and more robust.